op-chain-ops/genesis/layer_two.go

```diff
@@ -76,6 +76,9 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 	}
 
 	db, err = setupEcotone(config, db)
```

Comment:
Hiding the branching logic inside of `setupEcotone` (which sounds like it always applies ecotone) is a bit unclear

Reply:
Will change in base branch